packages/lesswrong/components/ea-forum/EADigestPage.tsx

```diff
@@ -0,0 +1,296 @@
+import moment from "moment";
+import { useLocation } from "@/lib/routeUtil";
+import { gql, useQuery } from "@apollo/client";
+import { useCurrentUser } from "../common/withUser";
+import { Link } from "@/lib/reactRouterWrapper";
+import { useMessages } from "../common/withMessages";
+import { useUpdateCurrentUser } from "../hooks/useUpdateCurrentUser";
+import { digestLink } from "./EABestOfPage";
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    display: 'grid',
+    gridTemplateColumns: '1fr 1fr',
+    justifyItems: 'end',
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    color: theme.palette.text.alwaysWhite,
+    padding: '30px 30px 60px',
+    marginTop: -30,
+    marginBottom: -165,
+    [theme.breakpoints.down("sm")]: {
+      display: 'block',
+      paddingTop: 50,
+      paddingLeft: 20,
+      paddingRight: 20,
+      marginLeft: -8,
+      marginRight: -8,
+    }
+  },
+  topSection: {
+    maxWidth: 765,
+    padding: '10px 3px 26px'
+  },
+  pageTitle: {
+    display: 'flex',
+    columnGap: '20px',
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontSize: 32,
+    fontWeight: 700,
+    marginTop: 0,
+    marginBottom: 6,
+  },
+  previewLabel: {
+    color: theme.palette.grey[300],
+    fontSize: 20,
+    fontWeight: 500,
+    marginLeft: 16,
+  },
+  subscribeTooltip: {
+  },
+  subscribeIcon: {
+    opacity: 0.7,
+    cursor: 'pointer'
+  },
+  success: {
+    display: 'flex',
+    alignItems: 'center',
+    columnGap: 10,
+    fontSize: 13,
+    lineHeight: '19px',
+    color: theme.palette.grey[800],
+    textWrap: 'pretty',
+  },
+  successCheckIcon: {
+    color: theme.palette.icon.greenCheckmark
+  },
+  successLink: {
+    color: theme.palette.primary.main
+  },
+  pageDescription: {
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontSize: 16,
+    fontWeight: 500,
+    lineHeight: "150%",
+  },
+  feedback: {
+    fontWeight: 500,
+    fontStyle: "italic",
+    fontSize: 14,
+    marginTop: 10,
+    opacity: 0.8,
+    "& a": {
+      textDecoration: 'underline',
+      '&:hover': {
+        textDecoration: 'underline',
+      }
+    },
+  },
+  heading: {
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontSize: 20,
+    fontWeight: 700,
+    marginTop: 0,
+    marginBottom: 20,
+  },
+  imageFade: {
+    background: `
+      linear-gradient(
+        90deg,
+        var(--digest-background) 0%,
+        var(--digest-background) 50%,
+        ${theme.palette.greyAlpha(0)} 80%
+      );
+    `,
+    position: "fixed",
+    top: 0,
+    left: 0,
+    width: '100%',
+    height: '100%',
+    zIndex: -1,
+    [theme.breakpoints.down("xs")]: {
+      background: `
+      linear-gradient(
+        135deg,
+        var(--digest-background) 0%,
+        var(--digest-background) 50%,
+        ${theme.palette.greyAlpha(0)} 80%
+      );
+    `,
+    }
+  },
+  image: {
+    position: "fixed",
+    top: 0,
+    right: 0,
+    width: '50%',
```

Comment:
I'm not sure this is what you want on mobile

Reply:
I think it looks fine on mobile? Is there something specific you'd change?
<img width="235" alt="Screen Shot 2024-06-13 at 7 44 56 PM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/807dce35-9d4b-494e-a041-1cb28f72dc60">